Retarget MSBuildSDKResolver

dsplaisted opened the pull request.

mmitche committed:
Do not sign test packages

mmitche committed:
Do not sign test packages take 2

dsplaisted committed:
Retarget MSBuildSDKResolver

dsplaisted committed:
Retarget templatelocator

rainersigwald reviewed (approved):
Makes sense to me. Should it be canonized as VSMacRuntimeVersion or something?

dsplaisted committed:
Don't retry when exception thrown in ExponentialRetry

This fixes issue where if you didn't set a hive in DotnetNewCommand,
it would throw a NullReferenceException instead of giving you the
correct error message.

dsplaisted committed:
Fix test

dsplaisted committed:
Add MSBuild property for resolver target framework

dsplaisted committed:
Skip ExponentialRetry test

dsplaisted commented:
> Makes sense to me. Should it be canonized as VSMacRuntimeVersion or something?

Good idea.  I went with `ResolverTargetFramework`.

@dotnet/dotnet-cli FYI I made some changes to fix some test failures.  There was an interaction with the exception thrown from `DotnetNewCommand` when you didn't specify a hive and the exponential retry code which would end up throwing a NullReferenceException.

dsplaisted committed:
Merge internal signing changes

mmitche commented:
LGTM on signing changes
